.github/workflows/ci-build.yml

```
@@ -208,223 +66,33 @@ jobs:
-        echo "::endgroup::"
-
-        rm -f opengfx-all.zip
-
-    - name: Install MSVC problem matcher
-      uses: ammaraskar/msvc-problem-matcher@master
-
-    - name: Configure developer command prompt for ${{ matrix.arch }}
-      uses: ilammy/msvc-dev-cmd@v1
-      with:
-        arch: ${{ matrix.arch }}
-
-    - name: Build
-      shell: bash
-      env:
-        NINJA_STATUS: "[%f/%t -- %e] " # [finished_edges/total_edges -- elapsed_time], default value is "[%f/%t] "
-      run: |
-        mkdir build
-        cd build
-
-        echo "::group::CMake"
-        cmake .. \
-          -GNinja \
-          -DVCPKG_TARGET_TRIPLET=${{ matrix.arch }}-windows-static \
-          -DCMAKE_TOOLCHAIN_FILE="${GITHUB_WORKSPACE}\vcpkg\scripts\buildsystems\vcpkg.cmake" \
-          # EOF
-        echo "::endgroup::"
-
-        echo "::group::Build"
-        cmake --build .
-        echo "::endgroup::"
-
-    - name: Test
-      shell: bash
-      run: |
-        cd build
-        ctest --timeout 120
-
-
-  msys2:
     strategy:
       fail-fast: false
       matrix:
         include:
-          - msystem: MINGW64
-            arch: x86_64
-          - msystem: MINGW32
-            arch: i686
-
-    name: MinGW (${{ matrix.arch }})
-
-    runs-on: windows-latest
-
-    steps:
-    - name: Checkout
-      uses: actions/checkout@v4
-
-    - name: Setup MSYS2
-      uses: msys2/setup-msys2@v2
-      with:
-        msystem: ${{ matrix.msystem }}
-        release: false
-        install: >-
-          git
-          make
-          mingw-w64-${{ matrix.arch }}-cmake
-          mingw-w64-${{ matrix.arch }}-gcc
-          mingw-w64-${{ matrix.arch }}-lzo2
-          mingw-w64-${{ matrix.arch }}-libpng
-          mingw-w64-${{ matrix.arch }}-lld
-          mingw-w64-${{ matrix.arch }}-ninja
-
-    - name: Install OpenGFX
-      shell: bash
-      run: |
-        mkdir -p "C:/Users/Public/Documents/OpenTTD/baseset"
-        cd "C:/Users/Public/Documents/OpenTTD/baseset"
-
-        echo "::group::Download OpenGFX"
-        curl -L https://cdn.openttd.org/opengfx-releases/0.6.0/opengfx-0.6.0-all.zip -o opengfx-all.zip
-        echo "::endgroup::"
-
-        echo "::group::Unpack OpenGFX"
-        unzip opengfx-all.zip
-        echo "::endgroup::"
-
-        rm -f opengfx-all.zip
-
-    - name: Install GCC problem matcher
-      uses: ammaraskar/gcc-problem-matcher@master
-
-    - name: Build
-      shell: msys2 {0}
-      env:
-        NINJA_STATUS: "[%f/%t -- %e] " # [finished_edges/total_edges -- elapsed_time], default value is "[%f/%t] "
-      run: |
-        mkdir build
-        cd build
+        - os: windows-latest
+          name: Windows
+          arch: x86
+        - os: windows-latest
+          name: Windows
+          arch: x64
 
-        echo "::group::CMake"
-        cmake .. \
-          -GNinja \
-          -DCMAKE_CXX_FLAGS="-fuse-ld=lld" \
-          # EOF
-        echo "::endgroup::"
+    name: ${{ matrix.name }} (${{ matrix.arch }})
```

Why not doing it like other targets?
```suggestion
    name: Windows (${{ matrix.arch }})
```

Of course also needs removal of `name` in the matrix.

---

Was when 2019 was still relevant :P can be removed now